### DISCUSSION TITLE
stop putting random shit in my dashboard

### DISCUSSION THREAD

**Discussion by donuts-are-good** [community/community#40278] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I dont care about anything social. I just want to see a list of my repos when I log in. I'm not interested in microsoft, social, or anything product-related. fucking stop.

**Comment by ossanna16** [comment]
Hi there @donuts-are-good and welcome to our community (love the username!). Thank you for your feedback!

**Comment by FrostyX** [comment]
Related discussion https://github.com/community/community/discussions/17120
The tone and formatting of the post could be better but I 100% agree with its point and sympathize with the frustration.
